SCREENSHOTS.md

```diff
@@ -0,0 +1,63 @@
+# Screenshot utilities
+
+This library provides several methods to faciliate implementing screenshot-based testing techniques.
+
+## Generate a screenshot
+
+It is possible to generate a PNG screenshot and save it into disk of the whole browser with the following method:
+
+```typescript
+const fileapath = 'my-folder/my-screenshot.png'
+await Browser.saveScreenshot(filepath);
+```
+
+It is also possible to get an in-memory base64 string with the taken screenshot as follows:
+
+```typescript
+const screenshotBase64: string = await Browser.takeScreenshot();
+```
+
+Similarly, an screenshot of the region occupied by a page or a widget can be taken using the analogous methods of `Widget` or `BasePage` classes.
```

Comment:
Suggestion: Could we add a piece of code of an example with that utility? I think could be great to put it here just to know how to do it.

Reply:
Completed SCREENSHOTS.md file with more examples. Available in next commit